data/parse_smogon_stats.py

```diff
@@ -41,64 +41,25 @@ def get_smogon_stats_file_name(game_mode, month_delta=1):
-                    segment = next(it)
-                    if ':' in segment:
-                        split_segment = segment.split()
-                        spread = split_segment[0]
-                        nature = normalize_name(spread.split(':')[0])
-                        evs = spread.split(':')[1].replace('/', ',')
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        pokemon_information[pokemon_name][SPREADS_STRING].append((nature, evs, percentage))
-
-            elif normalize_name(segment) == ITEM_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        item = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if item != OTHER_STRING:
-                            pokemon_information[pokemon_name][ITEM_STRING].append((item, percentage))
-
-            elif normalize_name(segment) == MOVES_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        move = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if move != OTHER_STRING:
-                            if constants.HIDDEN_POWER in move:
-                                move = "{}{}".format(move, constants.HIDDEN_POWER_ACTIVE_MOVE_BASE_DAMAGE_STRING)
-                            pokemon_information[pokemon_name][MOVES_STRING].append((move, percentage))
-
-            elif normalize_name(segment) == ABILITY_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        ability = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if ability != OTHER_STRING:
-                            pokemon_information[pokemon_name][ABILITY_STRING].append((ability, percentage))
-
-    return pokemon_information
+    infos = r.json()['data']
+    final_infos = {}
+    for x in infos.keys():
+        final_infos[x] = {}
+        for t in infos[x]['Spreads']:
+            spreads.append((t.split(':')[0], t.split(':')[1].replace('/', ','), float("{:.16f}".format(float(infos[x]['Spreads'][t])))))
+        for j in infos[x]['Items']:
+            items.append((j, infos[x]['Items'][j]))
+        for k in infos[x]['Moves']:
+            moves.append((k, infos[x]['Moves'][k]))
+        for l in infos[x]['Abilities']:
+            abilities.append((l, infos[x]['Abilities'][l]))
```

Comment:
The Chaos JSON is raw data - so there's quite a few times that Moves/Items/Abilities/Spreads are used so infrequently that the _percentage_ value is rounded to 0. A check should be put in place to omit any values that have a percentage value of 0.